src/admin/tests/test_smig.py

```diff
@@ -0,0 +1,127 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for smig.
+"""
+
+
+import os
+import unittest
+from unittest.mock import patch
+
+from lsst.qserv.admin.cli import script
+from lsst.qserv.qmeta.schema_migration import QMetaMigrationManager
+from lsst.qserv.schema.schemaMigMgr import SchemaUpdateRequired, Uninitialized
+
+
+latest_qmeta_schema_version = 4
```

Comment:
Why this number is here? Does it refer to the real version? Or this it's just for testing purposes? If not then it may become a management burden to update this variable here after updating the schema. I'd rather have it where the schema of the `qservMeta` database is defined.